packages/jazz-tools/src/coValues/coRichText.ts

```diff
@@ -615,7 +615,10 @@ export function splitNode(
  * Collection of predefined mark types for common text formatting.
  * Includes marks for headings, paragraphs, links, and text styling.
  */
-export const Marks = {
+export const Marks: Record<
+  "Heading" | "Paragraph" | "Link" | "Strong" | "Em",
+  typeof Mark
+> = {
```

Comment:
@bensleveritt 

Is this ok for you?
Another fix for the d.ts, probably related to the circular references.

Reply:
Yeah LGTM 👍